docs/deploy.sh

```diff
@@ -3,9 +3,36 @@ set -e
 
 BRANCH="gh-pages"
 
+current_time() {
+  
+  TIME=$(date -u --rfc-3339=seconds)
+
+  DATE=$(echo $TIME | cut -d' ' -f1)
+  HOUR=$(echo $TIME | cut -d' ' -f2)
+  
+  VALID_TIME=${DATE}T${HOUR}
+  
+  echo $VALID_TIME
+
+}
+
 build() {
   echo "Starting building..."
 
+  TIME=$(current_time)
+
+  printf "+++\ntitle = \"CHANGELOG\"\ndescription = \"Fang CHANGELOG\"\ndate = $TIME\nupdated = $TIME\ndraft = false\nweight = 410\nsort_by = \"weight\"\ntemplate = \"docs/page.html\"\n\n[extra]\nlead = \"Fang Changelog\"\ntoc = true\ntop = false\n+++\n\n" > docs/content/docs/CHANGELOG.md
+
+  cat CHANGELOG.md >> docs/content/docs/CHANGELOG.md
+
+  printf "+++\ntitle = \"README\"\ndescription = \"Fang README\"\ndate = $TIME\nupdated = $TIME\ndraft = false\nweight = 410\nsort_by = \"weight\"\ntemplate = \"docs/page.html\"\n\n[extra]\nlead = \"Fang README\"\ntoc = true\ntop = false\n+++\n\n" > docs/content/docs/README.md
```

Comment:
We have to delete description

Reply:
it think solve in [delete desc](https://github.com/ayrat555/fang/pull/82/commits/e3610e371047e58c899c7045843630aae2ea8740)